### DISCUSSION TITLE
Is it possible to renew an expired domain?

### DISCUSSION THREAD

**Discussion by ansonyuu** [vercel/community#2553] [Help] [answered]
### Summary

Bought a domain on Vercel and accidentally let it expire! Have tried turning auto-renewal on and off, but no luck

<img width="515" alt="image" src="https://github.com/vercel/community/assets/48683587/3cba248d-b407-41f0-9531-aaee851e3d4f">

<img width="518" alt="image" src="https://github.com/vercel/community/assets/48683587/d6877f8c-2434-4a74-951c-0d1b1659a8e5">

**Comment by amyegan** [comment, marked as answer]
Sorry you're having domain renewal trouble, @ansonyuu. There may be a problem with the payment method that the renewal was billed to, but it looks like the domain is still eligible to renew. If you [submit a case](https://vercel.com/help#issues) to the Customer Success Team under the "Billing" problem area, the team will be able to help you get this sorted.
